Octokit/Http/Connection.cs

```diff
@@ -524,6 +548,10 @@ public Credentials Credentials
             request.Headers.Add("User-Agent", UserAgent);
             await _authenticator.Apply(request).ConfigureAwait(false);
             var response = await _httpClient.Send(request, cancellationToken).ConfigureAwait(false);
+            if (response != null)
+            {
+                LastApiInfo = response.ApiInfo;
+            }
```

Comment:
@khellang I moved the setting of the LastApiInfo before HandleErrors so that the LastApiInfo is still set even if an exception is thrown from the HandleErrors